tools/df/docker/bird-build/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG BASE_IMAGE=276327358223.dkr.ecr.us-east-1.amazonaws.com/pipedream-dev
+ARG TAG=latest
+
+FROM ${BASE_IMAGE}:${TAG}
+
+RUN bash -c "apt-get update"
+# install build dependencies
+RUN bash -c "apt-get -y install build-essential autoconf flex bison git libncurses5-dev libreadline-dev debhelper" 
```

Comment:
I'm not sure you need to use bash -c, you should be able to get away with with just apt-get -y install like here
https://github.com/deepfield/deploy/blob/178c2528faa34ddf3cf3d29d058d8a340b336c0f/docker-images/base-build/Dockerfile#L42-L43

I also recommend crushing the RUN statements into 1 as it reduces layers, making the docker image smaller. 

Removing /var/lists will also remove the extra stuff that apt-get update pulls in. You won't need the contents of of this dir after you install your dependencies. Example:
https://github.com/deepfield/deploy/blob/178c2528faa34ddf3cf3d29d058d8a340b336c0f/docker-images/base-build/Dockerfile#L67